packages/gatsby/src/bootstrap/index.ts

```diff
@@ -0,0 +1,50 @@
+import { startRedirectListener } from "./redirects-writer"
+import {
+  IBuildContext,
+  initialize,
+  customizeSchema,
+  sourceNodes,
+  buildSchema,
+  writeOutRequires,
+  createPages,
+  createPagesStatefully,
+  extractQueries,
+} from "../services"
+import { Runner } from "./create-graphql-runner"
+import { writeOutRedirects } from "../services/write-out-redirects"
+import { postBootstrap } from "../services/post-bootstrap"
+import { rebuildWithSitePage } from "../schema"
```

Comment:
This one feels werid to actually not be service given that everything else is (other than types and one "startListener").

What qualifies something to be a service really?

Reply:
In short: it being used in state machine. I'm not 100% sure whether I'll need to use this one there, but you're right I should probably make it one anyway.

Reply:
I don't think this one will be used in any other place (schema-hot-reloader uses `rebuild` from `src/schema/index`), but I do like "entry points" for each ... "service" to be in same place (`src/services/`) vs some in services and some elsewhere.

Also another NIT - this change as is lost the `update schema` activity wrapping the execution of `rebuildWithSitePage`

```
activity = reporter.activityTimer(`update schema`, {	
    parentSpan: bootstrapSpan,	
  })
```

So if this is moved to service - that would be natural place to re-add it